File: tarea3/metodosArreglo.js
#!/usr/bin/env node

let arreglo = [0, 1, 2, 3, 4, 5];

const args = process.argv.slice(2);

if (args.length === 0) {
  console.log(arreglo);
} else {
  const comando = args[0];

  if (comando === "insertar") {
    arreglo.splice(+args[2], 0, +args[1]);
    console.log(`El nuevo arreglo es [${arreglo.join(", ")}]`);
  }

  else if (comando === "extraer") {
    arreglo.splice(+args[1], 1);
    console.log(`El nuevo arreglo es [${arreglo.join(", ")}]`);
  }
}

File: tarea3/metodosArreglo.py
#!/usr/bin/env python3
import typer

arreglo: list[int] = [0, 1, 2, 3, 4, 5]

app = typer.Typer(invoke_without_command= True)

@app.callback()
def main(ctx: typer.Context):
    if ctx.invoked_subcommand is None:
        print(arreglo)

@app.command()
def insertar(valor: int, indice: int):
    arreglo.insert(indice, valor)
    print(f"El nuevo arreglo es {arreglo}")


@app.command()
def extraer(indice: int):
    arreglo.pop(indice)
    print(f"El nuevo arreglo es {arreglo}")


if __name__ == "__main__":
    app()